src/interface/report/PlayerTile.tsx

```diff
@@ -103,6 +107,41 @@ const PlayerTile = ({ player, makeUrl, config }: Props) => {
       </span>
     );
   }
+  if (spec.id === SPECS.AUGMENTATION_EVOKER.id && isFightDungeon(fight)) {
+    return (
+      <span
+        className="player"
+        onClick={() => {
+          alert(
+            `Augmentation Evoker is not currently supported for M+ logs due to issues with retrieving the appropriate data for analysis. Augmentation is still supported for raid and we hope to re-enable it for M+ soon.`,
+          );
+        }}
+      >
+        <div className="role" />
```

Comment:
rather than duplicating the contents, this could be done by defining a parent component, then picking a parent based on the spec:

```tsx
const BlockLoading = (message) => ({ children }) => <span className="player" onClick={() => alert(message)}>{children}</span>;

let Container = ({ children}) => <Link {...}>{children}</Link>;

if (spec.id === SPECS.AUGMENTATION_EVOKER.id && isFightDungeon(fight)) {
  Container = BlockLoading(`...`);
}

return (
  <Container>
    <div className = "role" />
     <!-- ... -->
   </Container>
 );
 ```